.prettierrc.yml

```diff
@@ -1 +1,2 @@
 embeddedLanguageFormatting: off
+tabWidth: 2
```

Comment:
Remove

Reply:
See [a2f0d06](https://github.com/eecs-autograder/ag-website-vue/pull/534/commits/a2f0d063e478a418a34dc1b70101abf889c5c562). I deleted this file and we're just using the default config.

I'm not sure why `embeddedLanguageFormatting` was set to off, but that's why prettier wasn't running on code in script tags.